package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "html-pdf",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Comment:
We should do the versioning separately on master.
This will get messy if we have simultaneous pull requests open that change a version.
Maybe I'll set up [semantic-release](https://github.com/semantic-release/semantic-release) somewhen